sdk/compute/Azure.ResourceManager.Compute/CHANGELOG.md

```diff
@@ -1,9 +1,27 @@
 # Release History
 
-## 1.12.0-beta.1 (Unreleased)
+## 1.12.0 (2025-04-01)
```

Comment:
2025-04-01 is the api-version, not a release date since the date has passed.
Could we change this to possible release date?

Reply:
Is 2025-09-22 possible or is that too close? We want to push this sooner so we can build our Powershell features using .NET